tests/unit_tests/regression/test_rotor.cpp

```diff
@@ -62,7 +61,7 @@ TEST(RotorTest, IEA15Rotor) {
     constexpr size_t max_iter(6);
     constexpr double step_size(0.01);  // seconds
     constexpr double rho_inf(0.0);
-    constexpr double t_end(0.1);
+    constexpr double t_end(10);
```

Comment:
Same as above - looks like a change just for development.

Reply:
Or this